docs/machine-learning/how-to-guides/load-data-model-builder.md

```diff
@@ -28,42 +28,57 @@ Model Builder helps you create models for the following machine learning scenari
 - Issue classification (multiclass classification): Classify textual data into 3 or more categories.
 - Price prediction (regression): Predict a numeric value.
 - Image classification (deep learning): Categorize images based on characteristics.
+- Object detection (deep learning): Find objects within images. This can find one or more objects and label them accordingly.
```

Comment:
Let's reword these to align with the names in the tool.

Issue classification -> text classification
Price prediction -> Value prediction

- Remove sentiment analysis
- Add recommendation